test/integrations/destinations/linkedin_ads/processor/validationTestData.ts

```diff
@@ -0,0 +1,353 @@
+              context: {
+                traits: commonUserTraits,
+              },
+              timestamp: commonTimestamp,
+              messageId: 'a80f82be-9bdc-4a9f-b2a5-15621ee41df8',
+            }),
+            metadata: generateMetadata(1),
+            destination: overrideDestination(commonDestination, {
+              deduplicationKey: `properties.eventId`,
+            }),
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            error:
+              '[LinkedIn Conversion API] no matching conversion rule found for random event. Please provide a conversion rule. Aborting: Workflow: procWorkflow, Step: deduceConversionEventRules, ChildStep: undefined, OriginalError: [LinkedIn Conversion API] no matching conversion rule found for random event. Please provide a conversion rule. Aborting',
+            metadata: generateMetadata(1),
+            statTags: {
+              destinationId: 'default-destinationId',
+              errorCategory: 'dataValidation',
+              errorType: 'configuration',
+              implementation: 'cdkV2',
+              destType: 'LINKEDIN_ADS',
+              module: 'destination',
+              feature: 'processor',
+              workspaceId: 'default-workspaceId',
+            },
+            statusCode: 400,
+          },
+        ],
+      },
+    },
+  },
+  {
+    id: 'linkedin_ads-validation-test-3',
+    name: 'linkedin_ads',
+    description: '[Error]: Check for unsupported message type',
+    scenario: 'Framework',
+    successCriteria:
+      'Response should contain error message and status code should be 400, as we are sending a message type which is not supported by linkedin_ads destination and the error message should be Event type random is not supported',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            destination: commonDestination,
+            metadata: generateMetadata(1),
+            message: {
+              userId: 'user123',
+              type: 'random',
+              groupId: 'XUepkK',
+              traits: {
+                subscribe: true,
+              },
+              context: {
+                traits: {
+                  email: 'test@rudderstack.com',
+                  phone: '+12 345 678 900',
+                  consent: 'email',
+                },
+              },
+              timestamp: '2020-01-21T00:21:34.208Z',
+            },
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            error:
+              'message type random is not supported: Workflow: procWorkflow, Step: validateInput, ChildStep: undefined, OriginalError: message type random is not supported',
+            metadata: generateMetadata(1),
+            statTags: {
```

Comment:
We can create some common stat and import it for better readability

Reply:
done